lensless/benchmark.py

```diff
@@ -170,13 +240,28 @@ def benchmark(model, dataset, batchsize=1, metrics=None, **kwargs):
     assert isinstance(model._psf, torch.Tensor), "model need to be constructed with torch support"
     device = model._psf.device
 
+    def Reconstruction_Error(prediction, lensless):
+        convolver = model._convolver
+        if convolver.pad:
+            prediction = prediction.movedim(-3, -1)
+        else:
+            prediction = convolver._pad(prediction.movedim(-3, -1))
+            lensless = convolver._pad(lensless)
```

Comment:
why do you convolve the lensless measurement again with the PSF? or is this the original object?

Reply:
This seems to be an error. I will look into it.